.travis.yml

```diff
@@ -77,10 +77,14 @@ before_script:
   - git fetch https://github.com/OP-TEE/optee_os --tags
   - unset CC
 
+  # checkpatch.pl will ignore the following paths
+  - CHECKPATCH_IGNORE=$(echo core/lib/lib{fdt,tomcrypt} lib/lib{png,utee,utils,zlib})
```

Comment:
Well I've run checkpatch on some files in libutee and got some errors. Not many of them however. And I agree that this is mostly implementation code that we have control on. So, OK to not exclude libutee. I'll update.